server/src/main/java/io/crate/planner/statement/CopyFromPlan.java

```diff
@@ -171,9 +166,7 @@ public static BoundCopyFrom bind(AnalyzedCopyFrom copyFrom,
         final var settings = Settings.builder().put(properties).build();
 
         if (properties.contains("validation")) {
-            DEPRECATION_LOGGER.deprecatedAndMaybeLog(
-                "copy_from.validation",
-                "Using (validation = ?) in COPY FROM is no longer supported. Validation is always enforced");
+            throw new IllegalArgumentException("Setting 'validation' is not supported");
```

Comment:
What do you think about completely removing it?

Reply:
I added a test in https://github.com/crate/crate/pull/16477/commits/6f656c3ba7f61a8a7738970476ad1e46aa98b449 that validation is not supported anymore and had to add "temporal" exception because validation is added only in the last commit. 

Then I added validation and removed this (and kept test which is passing on both commits)

Reply:
Another way can be removing deprecation logger test and add  `test_validation_is_not_supported` with the validation but I wanted to group relevant changes (validation removal and test) together